pkg/collector/corechecks/cluster/ksm/kubernetes_state.go

```diff
@@ -859,19 +874,12 @@ func ownerTags(kind, name string) []string {
 //   - `phase` tag should be mapped to `pod_phase` on pod metrics only.
 //   - Ingress metrics have generic tag names (host/path/service_name/service_port).
 //     It's important to have them in a dedicated mapper override for ingresses.
-func labelsMapperOverride(metricName string) map[string]string {
-	if strings.HasPrefix(metricName, "kube_pod") {
-		return map[string]string{"phase": "pod_phase"}
-	}
-
-	if strings.HasPrefix(metricName, "kube_ingress") {
-		return map[string]string{
-			"host":         "kube_ingress_host",
-			"path":         "kube_ingress_path",
-			"service_name": "kube_service",
-			"service_port": "kube_service_port",
-		}
+func (k *KSMCheck) labelsMapperOverride(metricName string) map[string]string {
+	tok := strings.SplitN(metricName, "_", 3)
```

Comment:
could you add an example of the metrics name.

it could be useful to understand why we take `tok[1]` as resourceKind

Reply:
Comment added: https://github.com/DataDog/datadog-agent/pull/13973/commits/b99e2cc1327623c47860309e222d46cb26a43eae#diff-90fa541db8be1ce7c626a6d794c2d610578195443e97cd1579338f6ac2ea62df